cmake/OpenCVFindIPP.cmake

```diff
@@ -33,6 +33,9 @@ unset(IPP_VERSION_STR)
 unset(IPP_VERSION_MAJOR)
 unset(IPP_VERSION_MINOR)
 unset(IPP_VERSION_BUILD)
+unset(OPENCV_ICV_NAME)
+unset(OPENCV_ICV_HASH)
+unset(OPENCV_ICV_PLATFORM)
```

Comment:
OPENCV_ICV_NAME is used in the function scope only. Does this cleanup needed here?